rocAL/rocAL_pybind/examples/rocAL_example_doc/Classification_training_flowerdataset.ipynb

```diff
@@ -0,0 +1,745 @@
+{
```

Comment:
please remove all debug cells which are not needed from jupyter notebook. Applies to all